source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/activity/detail/PropertyActivityDetailContainer.tsx

```diff
@@ -0,0 +1,125 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useOrganizationRepository } from '@/features/contacts/repositories/useOrganizationRepository';
+import { usePersonRepository } from '@/features/contacts/repositories/usePersonRepository';
+import { usePropertyActivityRepository } from '@/hooks/repositories/usePropertyActivityRepository';
+import {
+  Api_PropertyActivity,
+  Api_PropertyActivityInvolvedParty,
+  Api_PropertyMinistryContact,
+} from '@/models/api/PropertyActivity';
+
+import { IPropertyActivityDetailViewProps } from './PropertyActivityDetailView';
+
+export interface IPropertyActivityDetailContainerProps {
+  propertyId: number;
+  propertyActivityId: number;
+  onClose: () => void;
+  View: React.FunctionComponent<React.PropsWithChildren<IPropertyActivityDetailViewProps>>;
+}
+
+export const PropertyActivityDetailContainer: React.FunctionComponent<
```

Comment:
The frontend test coverage doesn't seem to meet our minimal standards defined in our best practices. Is there at least a follow-up story for test coverage?